main/ruletester/ruletester.go

```diff
@@ -89,171 +84,146 @@ func ReadMetricsFile(file string) ([]string, error) {
 }
 
 func main() {
+	runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
This has always been configurable via the environment. Let's only override it if the environment variable isn't set.